tests/cephfs/cephfs_utilsV1.py

```diff
@@ -252,6 +254,27 @@ def enable_mds_logs(self, client, fs_name="cephfs", validate=True):
                         log.error("Unable to set the debug logs")
                         raise CommandFailed(f"Unable to set the debug logs : {out}")
 
+    def enable_mds_log_v1(self, client, validate=True, **kwargs):
```

Comment:
Hi @AmarnatReddy Why can't we use the existing method in same module enable_logs at https://github.com/red-hat-storage/cephci/blob/master/tests/cephfs/cephfs_utilsV1.py#L3843 , instead of adding new method just for mds?

Reply:
In previous builds, logging had to be manually enabled by fetching the MDS daemons and activating logs on all nodes. 
For now I am removing this code. will be raising new PR